runtimes/runtimes/webworker.ts

```diff
@@ -86,6 +86,7 @@ export const webworker = (props: RuntimeProps) => {
             exists: _path => Promise.resolve(false),
             getFileSize: _path => Promise.resolve({ size: 0 }),
             getTempDirPath: () => '/tmp',
+            getServerDataFolder: _serverName => '',
```

Comment:
nit:  What do you think of the name getServerDataDirPath for consistency with the other function names declared under fs?

Reply:
I used the "folder" word instead of "dir" to align with the LSP spec terminology. I plan to rename `getTempDirPath` to `getTempFolderPath` in the future changes for aligning our FS interface with Node.js FS interface. Does `getServerDataFolderPath` look good to you?

Reply:
This interface is not part of protocol, so we could go with `dir` here for consistency.